libraries/chain/asset_evaluator.cpp

```diff
@@ -598,7 +598,7 @@ void_result asset_publish_feeds_evaluator::do_apply(const asset_publish_feed_ope
 
 void_result asset_claim_fees_evaluator::do_evaluate( const asset_claim_fees_operation& o )
 { try {
-   FC_ASSERT( db().head_block_time() > HARDFORK_413_TIME );
+   FC_ASSERT( db().head_block_time() > HARDFORK_CORE_188_TIME );
```

Comment:
This change is wrong.

Reply:
Actually this line can be removed, since the hard fork time has passed.
But it's wrong to change it to 188.
If we decide to remove this line, we also need to update related unit test code.